src/qt/forms/createwalletdialog.ui

```diff
@@ -68,17 +68,33 @@
     <string>Encrypt Wallet</string>
    </property>
    <property name="checked">
-    <bool>true</bool>
+    <bool>false</bool>
+   </property>
+  </widget>
+  <widget class="QLabel" name="advanced_options_label">
+   <property name="geometry">
+    <rect>
+     <x>20</x>
+     <y>90</y>
+     <width>130</width>
+     <height>21</height>
+    </rect>
+   </property>
+   <property name="styleSheet">
+    <string notr="true">font-weight:bold;</string>
```

Comment:
(QT newb question: what is `notr`?)

Reply:
Sorry, should have looked it up. Per https://doc.qt.io/qt-5/designer-creating-custom-widgets.html, if the `notr` optional attribute is true, the value is not meant to be translated.

Reply:
I just copy-pasted that from another ui file :-)

Reply:
string is **no**t **tr**anslated :)